packages/ui-react/src/components/Account/Account.tsx

```diff
@@ -261,6 +267,10 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
           formSection({
             label: t('account.email'),
             onSubmit: async (values) => {
+              if (!(values.email && values.confirmationPassword)) {
```

Comment:
Minor for code readability (for me, at least):

```suggestion
              if (!values.email && !values.confirmationPassword) {
```